datacenter/ucp/2.1/guides/architecture.md

```diff
@@ -25,8 +25,11 @@ and Docker Compose.
 Docker UCP leverages the clustering and orchestration functionality provided
 by Docker.
 
+<a name="figure-architecture-2">
```

Comment:
IIRC, I did this for something I couldn't find in markdown, but don't remember what that was. Regardless, I removed my text linking to it. Will remove. Will also update the `architecture-1.svg` you mentioned.